.github/workflows/release.yml

```diff
@@ -17,7 +15,7 @@ jobs:
   release:
     name: Release package on NPM
     runs-on: ubuntu-latest
-    if: github.event.workflow_run.conclusion == 'success' && startsWith(github.event.workflow_run.head_branch, 'refs/tags/v')
+    if: github.event.workflow_run.conclusion == 'success' && github.ref_name == 'master' && startsWith(github.event.workflow_run.head_branch, 'v')
```

Comment:
What does it fix and how?

Reply:
Run this workflow only if the previous workflow (check) is successful
AND
If running on master
AND
If the head_branch starts with v (tag that has already been checked in check)